src/app/_components/NumberBadge.tsx

```diff
@@ -0,0 +1,13 @@
+import { BadgeWrapper } from '@/components/BadgeWrapper'
+
+export type NumberBadgeProps = {
+  number: number
+}
+
+export function NumberBadge({ number }: NumberBadgeProps) {
+  return (
+    <BadgeWrapper>
+      <span className="text-2xl font-bold">{number}</span>
+    </BadgeWrapper>
+  )
+}
```

Comment:
`IconBadge` and `NumberBadge` are quite similar - would it make sense to have one component - `Badge`?

Reply:
I did think of that – But I'm looking at creating more components, having a more modularized codebase. Also, if we had `Badge` it would effectively have two props and both would be optional. Just something I wanted to give a go.

Reply:
Actually! I just had an idea. Give me a moment ;-)

Reply:
Sometimes we do odd things. Thank you. Done!